data/json/proficiencies/chemistry.json

```diff
@@ -8,5 +28,65 @@
     "default_time_multiplier": 2,
     "default_fail_multiplier": 2.5,
     "time_to_learn": "6 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_mutagen",
+    "name": { "str": "Mutagen Production" },
+    "description": "Knowledge on the production of general mutagens.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_fail_multiplier": 2.5,
+    "time_to_learn": "12 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_serum",
+    "name": { "str": "Serum Production" },
+    "description": "Knowledge on the synthesization of the key to the perfect self.",
+    "can_learn": true,
+    "default_time_multiplier": 2.5,
+    "default_fail_multiplier": 5,
+    "time_to_learn": "6 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_fuel",
+    "name": { "str": "Fuel Production" },
+    "description": "Knowledge on how to produce and synthesize fuels, unfortunately this won't help make gasoline.",
+    "can_learn": true,
+    "default_time_multiplier": 2,
+    "default_fail_multiplier": 2.5,
+    "time_to_learn": "6 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_upper",
```

Comment:
As above. You could have a sub-proficiency of chemical synthesis, "pharmaceutical synthesis", representing your ability to get pure enough compounds for human consumption. That would cover this and the ones below, and would be appropriate for mutagens as well.

Reply:
Mutagens and serums do have a component that isn't synthesized - the extraction from various flesh/slime things. @I-am-Erk: What are your thoughts on making such extractions part of a biochemistry proficiency?